embedded/EscapeROOM/radio_morse_panel/radio_morse_panel.c

```diff
@@ -22,139 +48,84 @@ void unit()
 	}
 }
 
-void dot()
+void PlayDot()
 {
 	carrier = CARRIER_ON;
-	unit();
+	PlayUnit();
 	carrier = CARRIER_OFF;
 }
 
-void dash()
+void PlayDash()
 {
 	carrier = CARRIER_ON;
-	unit(); unit(); unit();
+	PlayUnit(); PlayUnit(); PlayUnit();
 	carrier = CARRIER_OFF;
 }
 
-void short_space()
+void PlayShortSpace()
 {
-	unit();
+	PlayUnit();
 }
 
-void letter_space()
+void PlayLetterSpace()
 {
-	unit(); unit(); unit();
+	PlayUnit(); PlayUnit(); PlayUnit();
 }
 
-void word_space()
+void PlayWordSpace()
 {
-	unit(); unit(); unit(); unit(); unit(); unit(); unit();
+	PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit();
 }
 
-void zero()
-{
-	dash(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void one()
-{
-	dot();  short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void two()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void three()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void four()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dash();
-}
-
-
-void five()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot();
-}
-
-void six()
-{
-	dash(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot();
-}
-
-void seven()
-{
-	dash(); short_space();
-	dash(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot();
+unsigned char GetLetterPositionInMorseAlphabet(char Letter, const MorseCode *MorseAlphabet, const unsigned char NumOfElements){
```

Comment:
- How about shorted function name?  findMorseChar?
- return pointer to char array (second structure element)? 
- maybe put whole morseCode structure array inside this function? Do you need it anywhere else? then only one argument "char letter" will be needed...